apps/settings/elements/languages.html

```diff
@@ -29,6 +29,11 @@ <h1 data-l10n-id="language-header"> Language </h1>
             </select>
           </span>
         </li>
+        <li>
+          <a class="menu-item" id="more-languages" href="#">
```

Comment:
we are trying not to use `id` in Settings app, so can we use "menuItem-more-languages" for example in `class` ?